tests/test-rest-tags-controller.php

```diff
@@ -305,7 +305,12 @@ public function test_get_items_search_args() {
 		$request = new WP_REST_Request( 'GET', '/wp/v2/tags' );
 		$request->set_param( 'search', 'App' );
 		$response = $this->server->dispatch( $request );
-		$this->assertEquals( 200, $response->get_status() );
+		$this->assertResponseStatus( 200, $response );
+		$this->assertResponseData( array(
+			array(
+				'name' => 'Apple',
+			),
+		), $response );
```

Comment:
Formatting here isn't right, I think. Newline before the first `array`, newline after `$response`